prowler/providers/kubernetes/services/kubelet/kubelet_config_yaml_ownership/kubelet_config_yaml_ownership.py

```diff
@@ -0,0 +1,25 @@
+from prowler.lib.check.models import Check, Check_Report_Kubernetes
+from prowler.lib.utils.utils import is_owned_by_root
+from prowler.providers.kubernetes.services.core.core_client import core_client
+
+
+class kubelet_config_yaml_ownership(Check):
+    def execute(self) -> Check_Report_Kubernetes:
+        findings = []
+        for node in core_client.nodes:
+            report = Check_Report_Kubernetes(self.metadata())
+            report.namespace = node.namespace
+            report.resource_name = node.name
+            report.resource_id = node.uid
+            # It can only be checked if Prowler is being executed inside a worker node
+            if node.inside:
+                report.status = "PASS"
+                report.status_extended = f"kubelet config.yaml file ownership is set to root:root in Node {node.name}."
+                if not is_owned_by_root("/var/lib/kubelet/config.yaml"):
```

Comment:
You have to check if this file is present, if not set a manual result since the file can be other with a different name.